app/code/Magento/CatalogGraphQl/etc/schema.graphqls

```diff
@@ -248,6 +248,8 @@ interface ProductInterface @typeResolver(class: "Magento\\CatalogGraphQl\\Model\
     id: Int @doc(description: "The ID number assigned to the product") @resolver(class: "Magento\\CatalogGraphQl\\Model\\Resolver\\Product\\EntityIdToId")
     name: String @doc(description: "The product name. Customers use this name to identify the product.")
     sku: String @doc(description: "A number or code assigned to a product to identify the product, options, price, and manufacturer")
+    quantity_and_stock_status: String @doc(description: "Stock status of the product")
```

Comment:
Let's rename this attribute to `is_in_stock` and change its type to `Boolean`.

Reply:
I've changed that to be is_in_stock and use a boolean.